web/src/Components/CommentsZone/CommentsZone.css

```diff
@@ -0,0 +1,56 @@
+.full-comments-zone {
+	border-top: 1px dashed #CCC;
+}
+
+.comments-zone {
+	height: 310px;
+	overflow: auto;
+}
+
+.comment {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
```

Comment:
You use `flex-direction` with a child having a `flex: 1` value, which means that this container's childrens will take as much place as possible to fit in the container's width. It that case, the `justify-contents` property is irrelevent.